bigtable/google/cloud/bigtable/client.py

```diff
@@ -252,3 +254,25 @@ def list_instances(self):
         instances = [
             Instance.from_pb(instance, self) for instance in resp.instances]
         return instances, resp.failed_locations
+
+    def list_clusters(self):
+        """List the clusters in the project.
+
+        :rtype: tuple
+        :returns:
+            (clusters, failed_locations), where 'clusters' is list of
+            :class:`google.cloud.bigtable.instance.Cluster`, and
+            'failed_locations' is a list of locations which could not
```

Comment:
What type is in the list of `failed_locations`?

Reply:
`failed_locations` contain `string`s as per [protobuf](https://github.com/googleapis/googleapis/blob/c89b6330b6386298f8ea65e47e0b77b28294e3e7/google/bigtable/admin/v2/bigtable_instance_admin.proto#L306)

Reply:
Can you please update the comment to reflect that?

Reply:
Done